build/run_parser.sh

```diff
@@ -3,17 +3,20 @@
 echo starting;
 cd /code
 # TODO: No params does not honor active-only semesters, this is hardcoded for now
-/usr/bin/python manage.py ingest jhu --term Fall --years 2021;
+# /usr/bin/python manage.py ingest jhu --term Fall --years 2021;
+# /usr/bin/python manage.py digest jhu;
+
+/usr/bin/python manage.py ingest jhu --term Fall --years 2020;
 /usr/bin/python manage.py digest jhu;
 
-#/usr/bin/python manage.py ingest jhu --term Spring --years 2020;
-#/usr/bin/python manage.py digest jhu;
+/usr/bin/python manage.py ingest jhu --term Spring --years 2020;
+/usr/bin/python manage.py digest jhu;
 
-#/usr/bin/python manage.py ingest jhu --term Fall --years 2019;
-#/usr/bin/python manage.py digest jhu;
-#
-#/usr/bin/python manage.py ingest jhu --term Spring --years 2019;
-#/usr/bin/python manage.py digest jhu;
+/usr/bin/python manage.py ingest jhu --term Fall --years 2019;
+/usr/bin/python manage.py digest jhu;
+
+/usr/bin/python manage.py ingest jhu --term Spring --years 2019;
+/usr/bin/python manage.py digest jhu;
 
```

Comment:
you can use one line to ingest multiple semesters ex: `/usr/bin/python manage.py ingest jhu --terms Spring Fall --years 2019 2020;`